### ISSUE TITLE
Text inputs on the Custom Styles page are missing programmatic labels

### ISSUE THREAD

**Issue by MichSpieler** [pressbooks/pressbooks#4436]
The text inputs on the Custom Styles page (Theme Web Styles, Your Web Styles) are are missing programmatic labels. 

<img width="1128" height="424" alt="Image" src="https://github.com/user-attachments/assets/f303ebc7-cdb6-462e-a2ba-d875f75ea1ac" />

**Comment by MichSpieler** [comment]
From discussion: WordPress has its specific version of Code Mirror; could be looked at together with the keyboard trap issue https://github.com/pressbooks/pressbooks/issues/2440
